### PR TITLE
Linters Setup

### PR DESCRIPTION
**Pull request by Kingobaino1**
**In this pull request**
- We setup linters for the HTML form project.

### CODE REVIEW

**Comment by Kingobaino1** [comment]
**In this project**
- we setup linters
- we cloned the INTUIT signup page

File: index.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Form Project</title>
+    <link href="CSS/style.css" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/b77bd6f2e2.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="container">
+        <header class="heading">
```
Comment (kevcap) [inline index.html]: the `<header>` tag can stay outside of the container
```
<body>
  <header></header>
  <main></main>
  <footer></footer>
</body>
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Form Project</title>
+    <link href="CSS/style.css" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/b77bd6f2e2.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="container">
```
Comment (kevcap) [inline index.html]: You could use the `<main>` tag as the container.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+                
+                <div class="data-input">
+                    <div class="e-mail">
+                        <label for="e-mail">Email address</label>
+                        <input id="e-mail" type="email" name="e-mail">
+                    </div>
+                    
+                    <div class="phone">
+                            <label for="phone">Phone (recommended)</label>
+                            <input id="phone" type="text" name="phone">
```
Comment (kevcap) [inline index.html]: - [ ] The type of the phone `<input>` should be `tel`, not text ([screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png)from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/)) 📞

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

### Hi team, you guys did a very good job so far!
#### I can see that you guys are working hard at this project, and you're almost done, there are just some required changes* that you guys have to implement in order to proceed:

_Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, in order to get more knowledge and have a better project to add at your portfolio.
 
### [REQUIRED CHANGES]
- [ ] Your code is throwing some warnings at the [HTML5 Validator](https://validator.w3.org/nu/#textarear), please fix them.💥
image
- [ ] Guys, next time be aware of some of your commits, they should be descriptive so the other programmer can easily understand what was changed. ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_commits.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
image
image
image
(If problem means linter errors, prefer to write like this: Fix linter errors or Fix general linter errors)

- [ ] The type of the phone `<input>` should be `tel `, not text ([screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png)from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/)) 📞
- [ ] Kindly remove all the unnecessary files (e.g. `package-lock.json`). 🗑
- [ ] Please remove all the unnecessarily repeated CSS code ([Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) Tip: Create one only class like this: .d-flex { display: flex } and use that same class when you need it. 
image
Explanation: You guys have a lot of repeated code, and it's not a good practice, remember to always keep your code DRY (Don't Repeat Yourself).

- [ ] Improve the title and description of this pull request. Since you guys aren't just setting up the linters, but building a project with forms, semantic HTML, and so on. For more info check [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)

### [OPTIONAL CHANGES]

- [ ] To improve UI/UX, add some padding to your inputs

## Summary
- Your code looks clean, and you're using the best HTML and CSS, just have to improve it a bit 👌
- Your repo looks very professional and well organized 🗃
- You both have an attention to details, keep it up! 🔝
- We just have to fix what was pointed above, but it will be easy for you 🔧💪

### But that was close 😅


### Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀

👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

  
<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Comment by kevcap** [comment]
> 
> 
> **In this pull request**
> 
>     * We setup linters for the HTML form project.

- [ ] Kindly add more description to this pull request! 😁

**Comment by abmAsadullah** [comment]
## INTUIT signup page ##
**In this project**
- We setup linter to have fast track the linting errors
- We cloned the INTUIT Sign Up page
- We created a form for Sign Up page

File: README.md [hunk 4]
```diff
@@ -1 +1,55 @@
-# HTML-Form_Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# EMBEDDING IMAGES AND VIDEO
```
Comment (kevcap) [inline README.md:3]: - [ ] Please update the name of this project! ( Project name: HTML Forms )

File: README.md [hunk 5]
```diff
@@ -1 +1,55 @@
...
+
+## Authors
+
+👤 **Author1**
+
+- Github: [@Kingobaino1](https://github.com/Kingobaino1)
+- Twitter: [@ibehkingso](https://twitter.com/ibehkingso)
+- Linkedin: [Kingsley](https://www.linkedin.com/in/ibeh-kingsley-obinna-568596177)
+
+👤 **Author2**
```
Comment (kevcap) [inline README.md:35]: - [ ] To make your readme file more professional, you guys can add your names here

File: README.md [hunk 6]
```diff
@@ -1 +1,55 @@
...
+[Live Demo Link](https://kingobaino1.github.io/HTML-Form_Project/.)
+
+
+## Getting Started
+
+**Log on to the Github and clone the repository to your local machine.**
+
+## Authors
+
+👤 **Author1**
```
Comment (kevcap) [inline README.md:29]: - [ ] To make your readme file more professional, you guys can add your names here

**Review by kevcap — APPROVED** [review]
# [STATUS] PROJECT APPROVED 🍾

### I really like your project! Congrats team, you guys did an excellent job!

_**Clean and modular code, following the best practices and the design looks great, keep it up!**_ 🥇

## [OPTIONAL CHANGES]
- [ ] To improve UI/UX, add some padding to your inputs.
- [ ] Replace Author1 and Author2 with your names.

This one should be required, but since it wasn't pointed out earlier 😅, I'll mark it as optional, but I 
strongly suggest that you guys implement it: 
- [ ] Update the project name in your README


## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

**Pull request merged**